packages/v4/patternfly-docs/content/get-started/design.md

```diff
@@ -63,14 +55,57 @@ The PatternFly symbol library contains individual components like buttons, alert
 
 #### Navigating the symbol library
-There are two kinds of symbols - library symbols and document symbols. Library symbols are available in any Sketch document, while document symbols are specific to the document in which they are found.
+There are 2 kinds of symbols - library symbols and document symbols. Library symbols are available in any Sketch document, while document symbols are specific to the document in which they are found.
 
 From the top toolbar in Sketch, select Insert → Symbols → Patternfly 4 to add symbols onto your page.
 
 PatternFly symbols are designed to be as flexible and adaptable as possible. As such, designers should avoid detaching symbols from the library. However, there are rare instances where this may be necessary. Just know that once a symbol is detached, that symbol will no longer automatically update with new releases.
 
 ![Symbol-library](./img/symbol-library.png)
 
+
+## Getting the design kit
+
+The PatternFly design kit was originally developed for [Sketch users](#sketch-users), but is now available for use through [any design tool](#using-other-design-tools).
+
+#### Download the font
+PatternFly currently uses Red Hat Text and Red Hat Display. To download these fonts, go to https://github.com/RedHatOfficial/RedHatFont and download the whole repository. Then select all files in the "TTF" folder and double-click. This will install them onto your computer locally.
+
+### Using Sketch
+
+#### Before you start designing
+The design kit is a collection of Sketch assets that make it easy for designers to create hi-fi mockups that accurately represent PatternFly components and layouts. It works with Sketch libraries. To use it, you need to install [Sketch version 68](https://www.sketch.com/) or above. 
+
+
+
+#### Download the template file
+1. Access the [PatternFly template file](https://www.sketch.com/s/729c2eee-e8b6-4fcd-8a79-f6faa8c30f89) on Sketch cloud.
+2. Click **Download Document** to open the file. Then, click File → Save as template.
+
+####  Add the symbol library
+3. Access the [PatternFly design library](https://www.sketch.com/s/2cf1063b-5283-4e0b-b8a6-cbb1ac07e29e) on Sketch cloud.
+5. Click **Add library to Sketch** on the bottom right to add the library. You'll be prompted with a modal. Click **Add library to Sketch** again. Sketch will show you the library in your preferences window.
+
+### Using other design tools
+
+#### Before you begin
+For non-Sketch users, we have created a compatible version of the symbol library to fit with any design tool of choice. We recommend importing the SVG versions into your design tool. Download the SVG set in the [patternfly-design-kit repo](https://github.com/patternfly/patternfly-design-kit/tree/master/Symbols-SVGs).
+
+
+
+### Looking for the Overpass design kit?
```

Comment:
@mcarrano Do you think we should still bother mentioning the overpass design kit here?
(This PR is to update the Get Started > Design page)

Reply:
No, I think at this point we can remove that. It's probably horribly out of date at this point, anyway.